src/app/clusters/groups-server/groups-server.cpp

```diff
@@ -148,7 +148,7 @@ bool emberAfGroupsClusterAddGroupCallback(app::CommandHandler * commandObj, cons
 
     response.groupId = commandData.groupId;
     response.status  = GroupAdd(fabricIndex, commandPath.mEndpointId, commandData.groupId, commandData.groupName);
-    err              = commandObj->AddResponseData(commandPath, response);
+    err              = commandObj->AddResponse(commandPath, response);
     if (CHIP_NO_ERROR != err)
```

Comment:
This is going to try to send a status response on failure.  Please remove the emberAfSendImmediateDefaultResponse call.